consensus/core/src/metrics.rs

```diff
@@ -109,6 +109,7 @@ pub(crate) struct NodeMetrics {
     pub(crate) leader_timeout_total: IntCounterVec,
     pub(crate) missing_blocks_total: IntCounter,
     pub(crate) missing_blocks_after_fetch_total: IntCounter,
+    pub num_of_bad_nodes: IntGauge,
```

Comment:
Is there a per-authority score metric?

Reply:
Each node will report how many bad nodes they calculated at the time of leader schedule swap

Reply:
I guess scores per host is reported from consensus handler, which would be useful.